src/switch_core_media.c

```diff
@@ -11859,16 +11942,26 @@ SWITCH_DECLARE(void) switch_core_media_gen_local_sdp(switch_core_session_t *sess
 		v_engine->rtcp_mux = 1;
 	}
 
-	if (!smh->mparams->rtcp_audio_interval_msec) {
-		smh->mparams->rtcp_audio_interval_msec = (char *)switch_channel_get_variable(session->channel, "rtcp_audio_interval_msec");
-		if (switch_channel_var_true(smh->session->channel, "force_rtcp_passthru")) {
-			smh->mparams->rtcp_audio_interval_msec = SWITCH_RTCP_AUDIO_PASSTHRU;
+	if (a_engine->rtcp_mux > 0) {
+		if (!smh->mparams->rtcp_audio_interval_msec) {
+			smh->mparams->rtcp_audio_interval_msec = (char *)switch_channel_get_variable(session->channel, "rtcp_audio_interval_msec");
+			if (force_rtcp_passthru_set) {
+				smh->mparams->rtcp_audio_interval_msec = SWITCH_RTCP_AUDIO_PASSTHRU;
+			}
 		}
+	} else {
+		if (!smh->mparams->partner_rtcp_audio_interval_msec) {
```

Comment:
We shouldn't process partner variable here since this is intended to be set for partner variable